dev-server/static/scripts/vitalsource-mosaic-book-element.js

```diff
@@ -98,4 +98,24 @@ export class MosaicBookElement extends HTMLElement {
     this.prevButton.disabled = index === 0;
     this.nextButton.disabled = index === this.chapterURLs.length - 1;
   }
+
+  getBookInfo() {
```

Comment:
Presumably `getBookInfo` exists on VitalSource's `<mosaic-book>` custom element?

Reply:
Yes. The `setChapter` method does not, I'll add a comment to call that out.